utils.py

```diff
@@ -453,3 +454,38 @@ def get_all_language_codes_and_names():
 def unescape_encoded_uri_component(escaped_string):
     """Unescape a string that is encoded with encodeURIComponent."""
     return urllib.unquote(escaped_string).decode('utf-8')
+
+
+CACHE_SLUG = None
+def get_cache_slug():
```

Comment:
Hm, you raise a good point. I thought about it, and I think it might still be better to get rid of this function as a separate thing, since we don't really want it to be used directly by external methods. The logic for grabbing the cache slug from the yaml file is simple enough that I think it's fine to replicate it in the test_utils.py file.

Also, the answer to your last question is yes.